### PR TITLE
Raster Layer from File Sample

### PR DESCRIPTION
**Pull request by ghost**
Demonstrates showing a raster layer from a local raster as an operational layer.

### CODE REVIEW

**Comment by ghost** [comment]
@john0005 Can you review this simple sample?

File: src/main/java/com/esri/samples/raster/raster_layer_file/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+<p>How to create and use a raster layer made from a local raster file.</p>
+
+<p><img src="RasterLayerFile.png"/></p>
+
+<h2>How it works</h2>
+
+<p>To add a `RasterLayer` as an operational layer from a local raster file:</p>
+<ol>
+  <li>Create a `Raster` from a raster file</li>
```
Comment (john0005) [inline src/main/java/com/esri/samples/raster/raster_layer_file/README.md]: Need to replace \`RasterLayer\`  with \<code>RasterLayer\</code>

File: src/main/java/com/esri/samples/raster/raster_layer_file/RasterLayerFileSample.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+      // add the map to a map view
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // add the raster as an operational layer
+      map.getOperationalLayers().add(rasterLayer);
+
+      // set viewpoint on the raster
+      rasterLayer.addDoneLoadingListener(() -> mapView.setViewpointGeometryAsync(rasterLayer.getFullExtent()));
```
Comment (john0005) [inline src/main/java/com/esri/samples/raster/raster_layer_file/RasterLayerFileSample.java]: I like it better when it  is a little zoomed out. `mapView.setViewpointGeometryAsync(rasterLayer.getFullExtent(), 150);`

**Review by john0005 — COMMENTED** [review]
Everything looks good just two little things

**Comment by ghost** [comment]
Addressed. Thanks. Merging.

**Pull request merged**